fuel/app/classes/materia/api/v1.php

```diff
@@ -68,6 +68,20 @@ static public function widget_instance_delete($inst_id)
 		return $inst->db_remove();
 	}
 
+	/**
+	 * @return bool, true if the current user can publish the given widget instance, false otherwise.
+	 */
+	static public function publish_verify($widget_id)
```

Comment:
rename to `widget_instance_publish_perms_verify` or `widget_instance_publish_perms_get` or anything that follows the existing convention better.